ansible_ai_connect/ai/api/wca/model_id_views.py

```diff
@@ -220,10 +221,12 @@ def do_validated_operation(request, api_key_provider, model_id_provider, on_succ
     event = None
     start_time = time.time()
     model_id = UNKNOWN_MODEL_ID
+
+    # An OrgId must be present
+    # See https://issues.redhat.com/browse/AAP-16009
+    organization = request._request.user.organization
+
     try:
-        # An OrgId must be present
-        # See https://issues.redhat.com/browse/AAP-16009
-        organization = request._request.user.organization
```

Comment:
No need to keep the two lines below in the try/except block.

Reply:
Ah, but there is. I originally moved these too but the finally block did not run when the User has no Organisation and tests that check Sement events had been created failed.